go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/golang/snappy v0.0.3
```

Comment:
leaving a comment here as a reminder to bump the version to `1.17` or `1.19` if we keep `atomic.Bool`

Reply:
I bumped the Go version to `1.19` and added it to the PR